tools/cli/commands/generate.js

```diff
@@ -510,6 +511,9 @@ async function createComposerJson( composerJson, answers ) {
 			composerJson.extra[ 'branch-alias' ] = composerJson.extra[ 'branch-alias' ] || {};
 			composerJson.extra[ 'branch-alias' ][ 'dev-trunk' ] = '0.1.x-dev';
 			composerJson.extra.textdomain = name;
+			composerJson.extra[ 'version-constants' ] = {
+				'::PACKAGE_VERSION': `src/class-${ answers.name }.php`,
```

Comment:
> If by default, we can add the first 0.1.0-alpha version in the package src/class-example.php skeleton as well.

Reminder if we want to get that added 👍

Reply:
Added it for now, erring on the side that it's more useful to have it, and if someone doesn't they can just delete it. Still open for discussion thought.